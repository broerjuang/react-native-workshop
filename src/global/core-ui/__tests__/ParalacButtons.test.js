import React from 'react';
import renderer from 'react-test-renderer';
import {ParallaxButtons} from '../index';

describe('Icon', () => {
  it('should render Icon corectly', () => {
    let component = renderer.create(
      <ParallaxButtons name={'md-star'} value={16} />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
