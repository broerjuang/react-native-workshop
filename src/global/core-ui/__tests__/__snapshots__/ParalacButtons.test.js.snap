// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render Icon corectly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "center",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "fontSize": 24,
          "fontWeight": "bold",
        }
      }
    >
       
      16
       
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "fontSize": 14,
        }
      }
    >
       
      md-star
       
    </Text>
  </View>
</View>
`;
